packages/web3-eth-accounts/src/account.ts

```diff
@@ -263,12 +385,79 @@ export const privateKeyToAddress = (privateKey: string | Buffer): string => {
 };
 
 /**
- * encrypt a private key given a password, returns a V3 JSON Keystore
- * https://github.com/ethereum/wiki/wiki/Web3-Secret-Storage-Definition
+ * encrypt a private key with a password, returns a V3 JSON Keystore
+ *
+ * Read more: https://github.com/ethereum/wiki/wiki/Web3-Secret-Storage-Definition
+ *
+ * @param privateKey - The private key to encrypt, 32 bytes.
+ * @param password - The password used for encryption.
+ * @param options - Options to configure to encrypt the keystore either scrypt or pbkdf2
```

Comment:
```suggestion
 * @param options - (optional) Options to configure to encrypt the keystore either scrypt or pbkdf2
```

Reply:
same as picture from above, optional is already included in docs so we dont need to write it